bundler.d/test.rb

```diff
@@ -13,4 +13,5 @@
   gem 'factory_girl_rails', '~> 4.5', :require => false
   gem 'rubocop-checkstyle_formatter', '~> 0.1'
   gem "poltergeist"
+  gem 'shoulda-matchers'
```

Comment:
I'd suggest either adopting 3.0.0-rc1 now or waiting until 3.0 final.. it might be easier than adopting 2.x, only to have to immediately change it.